usaspending_api/reporting/management/commands/populate_reporting_agency_overview.py

```diff
@@ -223,6 +223,10 @@ class TempTableName(Enum):
                 SUM(obligations_incurred_total_cpe) AS total_dollars_obligated_gtas
             FROM
                 gtas_sf133_balances AS gtas
+             INNER JOIN dabs_submission_window_schedule dabs ON
```

Comment:
If you end up needing to do another commit for something else, can you fix the tab alignment here?

Reply:
Did you find anything else I should address? I don't have plans to push another commit

Reply:
Nope, just making a note in case anything else came out of our discussion